examples/quickstart/tutorial/hadoop/docker/setup-zulu-repo.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Script to set up the Azul Zulu JDK yum repository.
+#
+
+# Hardcode GPG key so we don't have to fetch it over http.
```

Comment:
how can you be sure the one you got was legit ;) ?

Reply:
if we are using them only for jre , we can directly download same from https://cdn.azul.com/zulu/bin/zulu8.38.0.13-ca-jre8.0.212-linux_x64.tar.gz and not configure the yum repo.
Ref: https://www.azul.com/downloads/zulu/

Reply:
Do you mind if we keep the yum repo? That way, it'll always use the latest jdk.

Reply:
Ah, now I see why we are using yum there. sure, feel free to keep it in that case.